experimental/examples/bubblebench/baseline/package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "@fluid-experimental/bubblebench-baseline",
+  "name": "@fluid-example/bubblebench-baseline",
   "version": "0.59.3000",
+  "private": true,
```

Comment:
Are you sure this belongs in examples not experimental scope? I see a few hundred weekly downloads on npmjs (feels high...?), and it seems more about testing the tech not an example of how to build XYZ?  I don't actually have any context here just trying to get a sense.

https://www.npmjs.com/package/@fluid-experimental/bubblebench-sharedtree

Reply:
I don't believe the content needs to be published for any particular reason (i.e. I don't think we'd advise customers to take a dependency on the package for any reason), and the content looks like a demo to me.  @DLehenbauer looks like the likely owner and might have a position.